.travis.yml

```diff
@@ -62,6 +62,8 @@ jobs:
       language: cpp
       script:
         - cd firmware/boron-ins-fsm/test && g++ -std=c++11 -I../inc -I./ -I./mocks -o ConsoleTests consoleFunctionTests.cpp -lstdc++ -lm && ./ConsoleTests -s
+        - g++ -std=c++11 -I../inc -I./ -I./mocks -I../lib/CircularBuffer/src -o ins3331Tests ins3331Tests.cpp -lstdc++ -lm && ./ins3331Tests -s
+        - g++ -std=c++11 -I../inc -I./ -I./mocks -I./UnitTestLib -o DoorSensorTests imDoorSensorTests.cpp -lstdc++ -lm && ./DoorSensorTests -s
```

Comment:
I noticed that in previous failing tests, this already seems to be the case where it will run all the tests even if one fails, and even if only one fails, the build fails. I will try to get it in a single command that uses wildcards though. I agree that sounds better

Reply:
Awesome. That was what I was most worried about.

If you aren't making any headway with the wildcard version of the Travis command, feel free to leave it as it is now.